stepanblyschak/sonic-buildimage
# [teamd] Stop teamd after stopping swss in fast-reboot

Signed-off-by: Stepan Blyschak <stepanb@nvidia.com>



#### Why I did it

To optimize fast-reboot.  Teamd can be stopped after bgp is stopped and after swss is stopped because the last LACP packet can be sent still since syncd is still running. Saves 15 sec on shutdown.

#### How I did it

Defined in the manifest for teamd to be stopped after swss

#### How to verify it

Run it on the switch.

Before;

```
Mon 20 Jun 2022 03:53:07 PM UTC Saving counters folder before warmboot...
Mon 20 Jun 2022 03:53:10 PM UTC Prepare MLNX ASIC to fast-reboot: install new FW if required
Mon 20 Jun 2022 03:53:13 PM UTC Collecting logs to check ssd health before fast-reboot...
Mon 20 Jun 2022 03:53:13 PM UTC Stopping lldp.timer ...
Mon 20 Jun 2022 03:53:13 PM UTC Stopped lldp.timer ...
Mon 20 Jun 2022 03:53:13 PM UTC Stopping mgmt-framework.timer ...
Mon 20 Jun 2022 03:53:13 PM UTC Stopped mgmt-framework.timer ...
Mon 20 Jun 2022 03:53:13 PM UTC Stopping pmon.timer ...
Mon 20 Jun 2022 03:53:14 PM UTC Stopped pmon.timer ...
Mon 20 Jun 2022 03:53:14 PM UTC Stopping snmp.timer ...
Mon 20 Jun 2022 03:53:14 PM UTC Stopped snmp.timer ...
Mon 20 Jun 2022 03:53:14 PM UTC Stopping telemetry.timer ...
Mon 20 Jun 2022 03:53:14 PM UTC Stopped telemetry.timer ...
Mon 20 Jun 2022 03:53:14 PM UTC Stopping what-just-happened.timer ...
Mon 20 Jun 2022 03:53:14 PM UTC Stopped what-just-happened.timer ...
Mon 20 Jun 2022 03:53:14 PM UTC Stopping lldp ...
Mon 20 Jun 2022 03:53:15 PM UTC Stopped lldp
Mon 20 Jun 2022 03:53:15 PM UTC Stopping mux ...
Warning: The unit file, source configuration file or drop-ins of mux.service changed on disk. Run 'systemctl daemon-reload' to reload units.
Mon 20 Jun 2022 03:53:15 PM UTC Stopped mux
Mon 20 Jun 2022 03:53:15 PM UTC Stopping nat ...
Dumping conntrack entries failed
Warning: The unit file, source configuration file or drop-ins of nat.service changed on disk. Run 'systemctl daemon-reload' to reload units.
Mon 20 Jun 2022 03:53:15 PM UTC Stopped nat
Mon 20 Jun 2022 03:53:15 PM UTC Stopping radv ...
Mon 20 Jun 2022 03:53:16 PM UTC Stopped radv
Mon 20 Jun 2022 03:53:16 PM UTC Stopping sflow ...
Warning: The unit file, source configuration file or drop-ins of sflow.service changed on disk. Run 'systemctl daemon-reload' to reload units.
Mon 20 Jun 2022 03:53:17 PM UTC Stopped sflow
Mon 20 Jun 2022 03:53:17 PM UTC Stopping teamd ...
Mon 20 Jun 2022 03:53:18 PM UTC Stopped teamd
Mon 20 Jun 2022 03:53:18 PM UTC Stopping bgp ...
Mon 20 Jun 2022 03:53:23 PM UTC Stopped bgp
Mon 20 Jun 2022 03:53:23 PM UTC Stopping swss ...
Mon 20 Jun 2022 03:53:33 PM UTC Stopped swss
Mon 20 Jun 2022 03:53:33 PM UTC Stopping syncd ...
Mon 20 Jun 2022 03:53:43 PM UTC Stopped syncd
Mon 20 Jun 2022 03:53:43 PM UTC Stopping all remaining containers ...
Mon 20 Jun 2022 03:53:47 PM UTC Stopped all remaining containers ...
Warning: Stopping docker.service, but it can still be activated by:
  docker.socket
Mon 20 Jun 2022 03:53:48 PM UTC Enabling Watchdog before fast-reboot
Watchdog armed for 180 seconds
Mon 20 Jun 2022 03:53:48 PM UTC Rebooting with /sbin/kexec -e to SONiC-OS-202111.82-3dacfea43_Internal ...
```

After:

```
Tue 21 Jun 2022 03:35:34 PM UTC Saving counters folder before warmboot...
Tue 21 Jun 2022 03:35:38 PM UTC Prepare MLNX ASIC to fast-reboot: install new FW if required
Tue 21 Jun 2022 03:35:40 PM UTC Collecting logs to check ssd health before fast-reboot...
Tue 21 Jun 2022 03:35:41 PM UTC Stopping lldp.timer ...
Tue 21 Jun 2022 03:35:41 PM UTC Stopped lldp.timer ...
Tue 21 Jun 2022 03:35:41 PM UTC Stopping mgmt-framework.timer ...
Tue 21 Jun 2022 03:35:41 PM UTC Stopped mgmt-framework.timer ...
Tue 21 Jun 2022 03:35:41 PM UTC Stopping pmon.timer ...
Tue 21 Jun 2022 03:35:41 PM UTC Stopped pmon.timer ...
Tue 21 Jun 2022 03:35:41 PM UTC Stopping snmp.timer ...
Tue 21 Jun 2022 03:35:41 PM UTC Stopped snmp.timer ...
Tue 21 Jun 2022 03:35:41 PM UTC Stopping telemetry.timer ...
Tue 21 Jun 2022 03:35:41 PM UTC Stopped telemetry.timer ...
Tue 21 Jun 2022 03:35:41 PM UTC Stopping what-just-happened.timer ...
Tue 21 Jun 2022 03:35:41 PM UTC Stopped what-just-happened.timer ...
Tue 21 Jun 2022 03:35:41 PM UTC Stopping pmon ...
Tue 21 Jun 2022 03:35:44 PM UTC Stopped pmon
Tue 21 Jun 2022 03:35:44 PM UTC Stopping lldp ...
Tue 21 Jun 2022 03:35:46 PM UTC Stopped lldp
Tue 21 Jun 2022 03:35:46 PM UTC Stopping mux ...
Warning: The unit file, source configuration file or drop-ins of mux.service changed on disk. Run 'systemctl daemon-reload' to reload units.
Tue 21 Jun 2022 03:35:46 PM UTC Stopped mux
Tue 21 Jun 2022 03:35:46 PM UTC Stopping nat ...
Dumping conntrack entries failed
Warning: The unit file, source configuration file or drop-ins of nat.service changed on disk. Run 'systemctl daemon-reload' to reload units.
Tue 21 Jun 2022 03:35:47 PM UTC Stopped nat
Tue 21 Jun 2022 03:35:47 PM UTC Stopping radv ...
Tue 21 Jun 2022 03:35:47 PM UTC Stopped radv
Tue 21 Jun 2022 03:35:48 PM UTC Stopping sflow ...
Warning: The unit file, source configuration file or drop-ins of sflow.service changed on disk. Run 'systemctl daemon-reload' to reload units.
Tue 21 Jun 2022 03:35:48 PM UTC Stopped sflow
Tue 21 Jun 2022 03:35:48 PM UTC Stopping bgp ...
Tue 21 Jun 2022 03:35:53 PM UTC Stopped bgp
Tue 21 Jun 2022 03:35:53 PM UTC Stopping swss ...
Tue 21 Jun 2022 03:36:03 PM UTC Stopped swss
Tue 21 Jun 2022 03:36:03 PM UTC Stopping teamd ...
Tue 21 Jun 2022 03:36:04 PM UTC Stopped teamd
Tue 21 Jun 2022 03:36:04 PM UTC Stopping syncd ...
Tue 21 Jun 2022 03:36:10 PM UTC Stopped syncd
Tue 21 Jun 2022 03:36:10 PM UTC Stopping all remaining containers ...
Tue 21 Jun 2022 03:36:14 PM UTC Stopped all remaining containers ...
Warning: Stopping docker.service, but it can still be activated by:
  docker.socket
Tue 21 Jun 2022 03:36:15 PM UTC Enabling Watchdog before fast-reboot
Watchdog armed for 180 seconds
Tue 21 Jun 2022 03:36:15 PM UTC Rebooting with /sbin/kexec -e to SONiC-OS-202111.83-8d72c484f_Internal ...
```

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
